### ISSUE TITLE
MVP 3: 11 User Test

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#29]
The guiding belief behind Equalify is that people want an easy-to-use WCAG scan that scans any site.

MVP 3 tests that principle with a fully functional version of the app. Users can add their sites, and run them against the WAVE WCAG scan.

I'll look for missing features and general bugs. I'll also look for interest.

Every user will be onboarded during a 30-minute call. Users will share their screens as I help them set up the service and add a few sites. I'll schedule a follow-up call after one to two weeks of using it. Users are also invited to submit any bugs or issues to the repo's [issues](https://github.com/bbertucc/equalify/issues) tab.

By the end of this test, I hope to know what features users are interested in.

The MVP test is scheduled to commence on September 6, 2022. 

Let's see what happens!

**Comment by bbertucc** [comment]
The guiding belief behind Equalify is that people want an easy-to-use WCAG scan that scans any site.

MVP 1.2 tests that principle with a fully functional version of the app. Users can add their sites, and run them against the WAVE WCAG scan.

I'll look for missing features and general bugs. I'll also look for interest.

Every user will be onboarded during a 30-minute call. Users will share their screens as I help them set up the service and add a few sites. I'll schedule a follow-up call after one to two weeks of using it. Users are also invited submit any bugs or issues to the repo's [issues](https://github.com/bbertucc/equalify/issues) tab.

By the end of this test, I hope to know what features users are interested in and if Equalify has a life after the fact.

The MVP test is scheduled to commence September 6, 2022. 

Let's see what happens!

**Comment by bbertucc** [comment]
If you're interested, feel free to signup at https://calendly.com/bbertucc/equalify

**Comment by bbertucc** [comment]
We're at 18 users! I'm taking feedback until October 17. Then I'll start another development sprint around any issues.

**Comment by bbertucc** [comment]
Resolving this issue. Lessons learned and enhancements have been tagged in [Milestone 4](https://github.com/bbertucc/equalify/milestone/4).
